tests/serializers/test_model.py

```diff
@@ -565,6 +565,42 @@ def ser_x(self, v: Any, _) -> str:
     assert json.loads(s.to_json(Model(x=1000))) == {'x': '1_000'}
 
 
+def test_function_plain_field_serializer_with_computed_field():
+    @dataclasses.dataclass
+    class Model:
+        x: int
+
+        @property
+        def computed_field_x(self) -> int:
+            return self.x + 200
+
+        def ser_func(self, v: Any, serializer: core_schema.FieldSerializationInfo) -> str:
+            return str(v * 2)
```

Comment:
Perhaps we should do something with the info to verify that the correct data is present...

Reply:
sure, what should we test for ?

Reply:
Maybe just that you can extract the name of the field?

Reply:
ok. I have updated it.